display-posts-shortcode-remote.php

```diff
@@ -596,7 +603,7 @@ public static function shortcode( $untrusted, $content, $tag = 'display-posts-re
 				$html = $heading . $html;
 			}
 
-			$open  = "<{$atts['wrapper']} class=\"display-posts-listing\">" . PHP_EOL;
+			$open  = "<{$atts['wrapper']} class='display-posts-listing $wrapper_class'>" . PHP_EOL;
```

Comment:
If you want to avoid empty space in the output here, i.e. `class='display-posts-listing '`, you could just always make sure `display-posts-listing` is added to the array before imploding it. So at line 475 above:

```
if ( $atts['wrapper_class'] ) {
    if ( ! in_array( 'display-posts-listing', $atts['wrapper_class'] ) {
        $atts['wrapper_class'][] = 'display-posts-listing';
    }

    $wrapper_class = implode( ' ', $atts['wrapper_class'] );
}
```

And then this line you could just call:

```
$open = "<{$atts['wrapper']} class='$wrapper_class'>" . PHP_EOL;
```

Not a critical update, but it's nice not to have unnecessary white space in the html.

Reply:
Thank you for the suggestion I have modified it and now we don't get any empty space.